### ISSUE TITLE
`QClass` needs a `toString` method

### ISSUE THREAD

**Issue by pieterlexis** [PowerDNS/pdns#7389]
- Program: Authoritative, Recursor, dnsdist
 - Issue type: Feature request

### Description
In order to log the QClass ('IN', 'CH', 'HS'), it would be nice to have a `toString()` function on the `QClass` object, as well as having an `operator<<(QClass)` for the `Logger` that uses this function.

Once those are there, replacing all the logging of the QClass to use this would be handy as well.

**Comment by pieterlexis** [comment]
reopening, as #10245 did not add a `operator<<(QClass)` in the logger

**Comment by omoerbeek** [comment]
I have something brewing, but it depends on #10282
